webapi/src/main/scala/org/knora/webapi/store/triplestore/impl/TriplestoreServiceLive.scala

```diff
@@ -437,226 +393,103 @@ case class TriplestoreServiceLive(
-   *
-   * @param outputFile        the output file.
-   * @param graphIri           the IRI of the graph used in the output.
-   * @param quadFormat         the output format.
-   * @param response           the response to be read.
-   * @return a [[Unit]].
-   */
-  private def writeResponseFileAsTurtleContent(
-    outputFile: Path,
-    graphIri: IRI,
-    quadFormat: QuadFormat,
-  )(response: CloseableHttpResponse): Task[Unit] =
-    Option(response.getEntity) match {
-      case Some(responseEntity: HttpEntity) =>
-        ZIO.attemptBlocking {
-          // Yes. Stream the HTTP entity to a temporary Turtle file.
-          val tempTurtleFile = Paths.get(outputFile.toString + ".ttl")
-          Files.copy(responseEntity.getContent, tempTurtleFile, StandardCopyOption.REPLACE_EXISTING)
-
-          RdfFormatUtil.turtleToQuadsFile(
-            RdfInputStreamSource(new BufferedInputStream(Files.newInputStream(tempTurtleFile))),
-            graphIri,
-            outputFile,
-            quadFormat,
-            StandardOpenOption.APPEND,
-          )
-
-          Files.delete(tempTurtleFile)
-          ()
-        }
-
-      case None =>
-        val message = s"Triplestore returned no content for graph $graphIri"
-        val error   = TriplestoreResponseException(message)
-        ZIO.logError(error.toString) *>
-          ZIO.fail(error)
-    }
 }
 
 object TriplestoreServiceLive {
-  private def makeHttpClient(config: Triplestore, host: HttpHost) =
-    ZIO.acquireRelease {
-      val connManager = new PoolingHttpClientConnectionManager()
-      connManager.setDefaultSocketConfig(SocketConfig.custom().setTcpNoDelay(true).build())
-      connManager.setValidateAfterInactivity(1000)
-      connManager.setMaxTotal(100)
-      connManager.setDefaultMaxPerRoute(15)
-
-      val credentialsProvider = new BasicCredentialsProvider
-      credentialsProvider.setCredentials(
-        new AuthScope(host.getHostName, host.getPort),
-        new UsernamePasswordCredentials(config.fuseki.username, config.fuseki.password),
-      )
-
-      // the client config used for queries to the triplestore. The timeout has to be larger than
-      // tripleStoreConfig.queryTimeoutAsDuration and tripleStoreConfig.gravsearchTimeoutAsDuration.
-      val requestTimeoutMillis = 7_200_000 // 2 hours
-      val requestConfig = RequestConfig
-        .custom()
-        .setConnectTimeout(requestTimeoutMillis)
-        .setConnectionRequestTimeout(requestTimeoutMillis)
-        .setSocketTimeout(requestTimeoutMillis)
-        .build
-
-      val httpClient: CloseableHttpClient = HttpClients
-        .custom()
-        .setConnectionManager(connManager)
-        .setDefaultCredentialsProvider(credentialsProvider)
-        .setDefaultRequestConfig(requestConfig)
-        .build()
-      ZIO.succeed(httpClient)
-    }(client => ZIO.attemptBlocking(client.close()).logError.ignore)
+  import scala.concurrent.duration._
```

Comment:
nitpick: I prefer imports at the start of the file

Reply:
This is solely due to an implicit config, because the default is zio duration convenience methods, but I need the finite duration ones and I couldn't find a conversion to FiniteDuration specifically.

Reply:
fair enough, didn't consider that